test/verify/check-system-info

```diff
@@ -421,6 +425,65 @@ class TestSystemInfo(MachineCase):
 
         self.machine.execute("umount /sys/class/dmi")
 
+    def testCPUSecurityMitigations(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go('/system/hwinfo')
+        b.enter_page('/system/hwinfo')
+        b.wait_present('#hwinfo a:contains(Mitigations)')
+        b.click('#hwinfo a:contains(Mitigations)')
+        b.wait_present('#cpu-mitigations-dialog tr:contains(No mitigations available)')
```

Comment:
As discussed above, this should check  that it's *not* present, as our VMs have no SMT.